images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -34,7 +34,7 @@ Expand-Archive -Path .\android-sdk-licenses.zip -DestinationPath 'C:\Program Fil
 # run the updates.
 # keep newer versions in descending order
 
-$sdk_root = "C:\Program Files (x86)\Android\android-sdk"
+$sdk_root = "C:\Progra~2\Android\android-sdk"
```

Comment:
You can break some programs what don't use 8.3 short path. Can we use a separate folder at the root of C:\ without space?

Reply:
I agree with you. Thank you. PR has been re-worked.